### ISSUE TITLE
Indicators have wrong spacing after screen unlock in GNOME 3.34

### ISSUE THREAD

**Issue by shibotto** [hardpixel/unite-shell#132]
Mouse hovering restores the correct spacing. Here's a brief demo:

![unite](https://user-images.githubusercontent.com/3816409/66110015-27786e00-e5c6-11e9-9e7c-2b98df9df29e.gif)

**Comment by jonian** [comment]
Hi @shibotto, thanks for the report.  
Please give more info about your system and the output of `journalctl /usr/bin/gnome-shell -r`

**Comment by jonian** [comment]
@shibotto are you using the master branch (can reproduce) or the stable version (can't reproduce)?

**Comment by shibotto** [comment]
Sorry for the delay, but for now I can't reproduce this anymore. All I did was disabling the user themes extension and then re-enabling it. To be absolutely sure I did the exact same steps on my other PC too, with the same result. I'm closing this until I'm able to reproduce it again (hoping I won't 😅)

I'm currently using the extensions.gnome.org version on Gentoo w/o systemd, so I can't give you journalctl output.
